modules/operators-overview.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='operators-overview-{context}']
+= Operators in {product-title}
+
+{product-title} v4 uses different classes of Operators to perform cluster 
+operations and run services on the cluster for your applications to use.
+
+[id='platform-operators-{context}']
+== Platform Operators in {product-title}
+
+In {product-title} version 4.0, all cluster functions are divided into a series
```

Comment:
> ... the first is to differentiate the 3.x and 4.x streams, and the second is specifically discussing the 4.0 cluster topology, which I imagine will evolve through the releases.

Do you foresee the Operator approach changing over the life of 4.x?  If we switch to something else, I'd expect that would trigger a bump to OpenShift 5.  Also, down in the `v4` paragraph below you're talking about the `kubelet` being run by systemd instead of as a possibly containerized process, and that seems much more mutable to me than "we're going to use Operators".